<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PitchTimbreFrame</class>
 <widget class="QFrame" name="PitchTimbreFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
background-color: rgb(64, 64, 64); 
color: rgb(208, 208, 208);
}


QComboBox:disabled {
  background-color: rgb(96, 96, 96); 
}

QSpinBox:disabled {
  background-color: rgb(96, 96, 96); 
}</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::Panel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Plain</enum>
  </property>
  <property name="lineWidth">
   <number>1</number>
  </property>
  <widget class="QComboBox" name="WaveFormComboBox">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>8</y>
     <width>120</width>
     <height>24</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <item>
    <property name="text">
     <string>Square</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Triangle</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Saw</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="DutyCycleComboBox">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>148</x>
     <y>8</y>
     <width>156</width>
     <height>24</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>4</number>
   </property>
   <item>
    <property name="text">
     <string>Duty Cycle 50</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Duty Cycle 25</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Duty Cycle 12.5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Duty Cycle 75</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>No Duty Cycle</string>
    </property>
   </item>
  </widget>
  <widget class="QSpinBox" name="AttackTimeSpinBox">
   <property name="geometry">
    <rect>
     <x>204</x>
     <y>40</y>
     <width>64</width>
     <height>24</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="maximum">
    <number>9999</number>
   </property>
   <property name="singleStep">
    <number>10</number>
   </property>
   <property name="value">
    <number>20</number>
   </property>
  </widget>
  <widget class="QLabel" name="msLabel">
   <property name="geometry">
    <rect>
     <x>276</x>
     <y>40</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>ms</string>
   </property>
  </widget>
  <widget class="QComboBox" name="AttackCurveComboBox">
   <property name="geometry">
    <rect>
     <x>74</x>
     <y>40</y>
     <width>128</width>
     <height>24</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Linear</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Exponential</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Gaussian</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Fermi</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="AttackLabel">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>40</y>
     <width>64</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Attack</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="DecayTimeSpinBox">
   <property name="geometry">
    <rect>
     <x>204</x>
     <y>70</y>
     <width>64</width>
     <height>24</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="maximum">
    <number>9999</number>
   </property>
   <property name="singleStep">
    <number>10</number>
   </property>
   <property name="value">
    <number>10</number>
   </property>
  </widget>
  <widget class="QComboBox" name="DecayCurveComboBox">
   <property name="geometry">
    <rect>
     <x>74</x>
     <y>70</y>
     <width>128</width>
     <height>24</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <item>
    <property name="text">
     <string>Linear</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Exponential</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Gaussian</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Fermi</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="DecayLabel">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>70</y>
     <width>64</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Decay</string>
   </property>
  </widget>
  <widget class="QLabel" name="msLabel_2">
   <property name="geometry">
    <rect>
     <x>276</x>
     <y>70</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>ms</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="SustainLevelSpinBox">
   <property name="geometry">
    <rect>
     <x>132</x>
     <y>100</y>
     <width>64</width>
     <height>24</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>127</number>
   </property>
   <property name="singleStep">
    <number>8</number>
   </property>
   <property name="value">
    <number>96</number>
   </property>
  </widget>
  <widget class="QLabel" name="ReleaseLabel">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>130</y>
     <width>64</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Release</string>
   </property>
  </widget>
  <widget class="QComboBox" name="ReleaseCurveComboBox">
   <property name="geometry">
    <rect>
     <x>74</x>
     <y>130</y>
     <width>128</width>
     <height>24</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <item>
    <property name="text">
     <string>Linear</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Exponential</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Gaussian</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Fermi</string>
    </property>
   </item>
  </widget>
  <widget class="QSpinBox" name="ReleaseTimeSpinBox">
   <property name="geometry">
    <rect>
     <x>204</x>
     <y>130</y>
     <width>64</width>
     <height>24</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="maximum">
    <number>9999</number>
   </property>
   <property name="singleStep">
    <number>10</number>
   </property>
   <property name="value">
    <number>30</number>
   </property>
  </widget>
  <widget class="QLabel" name="msLabel_3">
   <property name="geometry">
    <rect>
     <x>276</x>
     <y>130</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>ms</string>
   </property>
  </widget>
  <widget class="QLabel" name="LevelLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>100</y>
     <width>41</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Level</string>
   </property>
  </widget>
  <widget class="QLabel" name="SustainLabel">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>100</y>
     <width>64</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Sustain</string>
   </property>
  </widget>
  <widget class="QLabel" name="DamperOnButNoteOffSustainLabel">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>170</y>
     <width>212</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Sustain as damper on but note off</string>
   </property>
  </widget>
  <widget class="QComboBox" name="DamperOnButNoteOffSustainCurveComboBox">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>200</y>
     <width>128</width>
     <height>24</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Linear</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Exponential</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Gaussian</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Fermi</string>
    </property>
   </item>
  </widget>
  <widget class="QSpinBox" name="DamperOnButNoteOffSustainSustainLevelSpinBox">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>200</y>
     <width>58</width>
     <height>24</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>127</number>
   </property>
   <property name="singleStep">
    <number>8</number>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLabel" name="StartLevelLabel">
   <property name="geometry">
    <rect>
     <x>144</x>
     <y>200</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Level</string>
   </property>
  </widget>
  <widget class="QLabel" name="SecLabel">
   <property name="geometry">
    <rect>
     <x>284</x>
     <y>170</y>
     <width>32</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>sec</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="DamperOnButNoteOffSustainTimeDoubleSpinBox">
   <property name="geometry">
    <rect>
     <x>224</x>
     <y>170</y>
     <width>54</width>
     <height>25</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="decimals">
    <number>1</number>
   </property>
   <property name="maximum">
    <double>99.900000000000006</double>
   </property>
   <property name="singleStep">
    <double>0.500000000000000</double>
   </property>
   <property name="value">
    <double>8.000000000000000</double>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
